src/runners/violation_suppressions_runner.py

```diff
@@ -11,7 +11,6 @@
 )
 from runners.helpers import db, log
 
-
```

Comment:
I think separating imports from declarations used in the file with double blank lines makes sense, to visually separate it from the import block, similar to how functions in a file are separated, and unlike how different kinds of imports are separated. we can revisit these kinds of things but let's not add them to PR's unless they're standardizing on a new thing we're trying to do.

Reply:
fixed